libs/dh/admin/feature-user-management/src/lib/tabs/roles-tab/dh-roles-tab.component.ts

```diff
@@ -51,12 +64,16 @@ import {
     PushModule,
     DhRolesTabTableComponent,
     DhSharedUiPaginatorComponent,
+    WattButtonModule,
     DhRolesTabListFilterComponent,
     DhTabDataGeneralErrorComponent,
     LetModule,
   ],
 })
 export class DhUserRolesTabComponent {
+  @Input() roles: UserRoleDto[] = [];
```

Comment:
suggestion: seems unused. Can be deleted.

Reply:
Will do